feeds/src/components/aggregatorVis/AggregatorVis.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Vis } from '../vis'
+import { FeedConfig } from 'feeds'
+
+interface StateProps {
+  config: FeedConfig
+  latestOraclesState: any
+  latestAnswerTimestamp: any
+  latestAnswer: any
+  pendingAnswerId: any
+  oracleAnswers: any
+  oracleList: any
+  latestRequestTimestamp: any
+  minimumAnswers: any
+}
+
+interface DispatchProps {
+  fetchJobId: any
+}
+
+export interface Props extends StateProps, DispatchProps {}
+
+const AggregatorVis: React.FC<Props> = props => <Vis {...props} />
```

Comment:
Can you combine `.component` with `.enhanced` e.g. https://github.com/smartcontractkit/chainlink/blob/develop/feeds/src/components/listing/Listing.tsx